<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(247, 247, 247);</string>
  </property>
  <widget class="QTextEdit" name="MessageEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>481</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0;
font: 12pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessage">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>570</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(70, 88, 255);
color: rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChineseChess">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(70, 88, 255);
color: rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>象棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GoBang">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>570</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(70, 88, 255);
color: rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>五子棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Go">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(70, 88, 255);
color: rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>围棋</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="MessageBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top:0;
background-color: rgb(245, 245, 245);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Snake">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>570</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-color: rgb(70, 88, 255);
color: rgb(255, 255, 255);
font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>贪吃蛇</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
